tests/unit/integrations/airflow/orchestrators/test_dag_generator.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
```

Comment:
I feel we could flesh this out a bit, with all the experience from source_utils I think especially functions like import_class_by_path() might need to get some testing love

Reply:
Good point. I actually copied that function from `source_utils` (unfortunately that's necessary), but I now added tests for both this and the original.